spec/intl.html

```diff
@@ -1685,20 +1685,64 @@ <h1>
           </dl>
         </emu-clause>
 
-        <emu-clause id="sec-temporal-calendardatemergefields" type="abstract operation">
+        <emu-clause id="sec-temporal-calendarmutuallyexclusivekeys" type="abstract operation">
           <h1>
-            CalendarDateMergeFields (
+            CalendarMutuallyExclusiveKeys (
               _calendar_: a String,
-              _fields_: an Object,
-              _additionalFields_: an Object,
-            ): an Object
+              _key_: a property key,
+            ): a List of property keys
           </h1>
           <dl class="header">
             <dt>description</dt>
-            <dd>It takes two Objects of calendar-specific fields for the calendar represented by _calendar_ in _fields_ and _additionalFields_ and returns a new Object that includes both sets of fields. The values in _additionalFields_ should supersede the values in _fields_. Also, the returned Object must be free of ambiguity or conflicts. This is relevant for calendars which accept fields other than the standard set of built-in calendar fields.</dd>
+            <dd>
+              It performs the calendar-specific operation of determining which fields are mutually exclusive for the field given by _key_.
+              The returned List may be empty.
+            </dd>
           </dl>
+          <p>
+            Mutually exclusive means that the fields are alternate, potentially conflicting, ways of expressing the same data.
```

Comment:
Is this sentence (and the name of this AO) accurate if `day` causes exclusion of `era` and `eraYear` in Japanese?  `day` isn't an alternate way to specify the year, but it's returned nontheless.

Reply:
This is a good point. I think renaming it to CalendarKeysToExclude and adjusting the description a bit should be sufficient.